shared/src/protocol/account.rs

```diff
@@ -11,18 +11,21 @@ pub struct Account {
 }
 
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+#[allow(clippy::module_name_repetitions)]
```

Comment:
I think this exception should be removed since we also ended up exposing `navigation` and `pilot` in `protocol` for similar reasons.

Reply:
Done.